src/App.js

```diff
@@ -250,6 +257,9 @@ class App extends Component {
                 <div>3. Now you can deploy the contract in the Run tab!</div>
               </Popup>
             </Header.Content>
+            <a href="https://github.com/LayerXcom/vyper-remix" target="_blank" align="right" style={{"text-decoration": "none"}}>
```

Comment:
Despite `style={{"text-decoration": "none"}}`,  why the color still changes sky blue without mouse over...

![image](https://user-images.githubusercontent.com/22876645/49995917-e2e8fb00-ffd0-11e8-9b4e-9cd379184410.png)